ec2/sc-ec2-windows-jumpcloud.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'Service Catalog: Windows EC2 with Jumpcloud integration.'
+Description: 'Service Catalog: Windows EC2'
```

Comment:
We actually need to keep Jumpcloud on the Windows machines. Please leave the connector on this product.

Reply:
done